src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1334,6 +1334,7 @@ class DocBaseViewer extends BaseViewer {
             this.cachePage(pageNumber);
         }
 
+        this.renderUI();
```

Comment:
We should probably move this up to where the prior code was called, just to be safe.